src/main/java/com/mongodb/spark/sql/connector/connection/DefaultMongoClientFactory.java

```diff
@@ -38,12 +42,14 @@ public final class DefaultMongoClientFactory implements MongoClientFactory {
    */
   public DefaultMongoClientFactory(final MongoConfig config) {
     this.config = config;
+    this.mongoDriverInformation =
+        generateMongoDriverInformation(config instanceof ReadConfig ? "Source" : "Sink");
```

Comment:
Let's make these lowercase for consistency with the driver (which uses `"sync"`).